### PR TITLE
feat(Tag): add Tag to @carbon/styles

### PR DESCRIPTION
**Pull request by dakahn**
Adds Tag to @carbon/styles

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

**Comment by tw15egan** [comment]
The styles don't seem to be rendering correctly 🤔 

Screen Shot 2021-07-08 at 1 29 14 PM

**Review by andreancardona — DISMISSED** [review]
also not seeing updated styles :(

**Comment by dakahn** [comment]
Sorry this was in limbo so long team -- this one was a real doozy

File: packages/carbon-react/.storybook/styles.scss [hunk 1]
```diff
@@ -9,12 +9,16 @@ $feature-flags: (
...
+@use 'sass:meta';
 @use "~@ibm/plex/default" as sans;
 @use "~@ibm/plex/mono";
 @use "~@ibm/plex/arabic";
 
 @use '../index.scss' as styles;
 @use '../scss/components/button';
+@use '../scss/components/tag';
+
+@debug meta.module-variables('tag');
```
Comment (tw15egan) [inline packages/carbon-react/.storybook/styles.scss]:
```suggestion
```

File: packages/carbon-react/.storybook/styles.scss [hunk 2]
```diff
@@ -9,12 +9,16 @@ $feature-flags: (
   enable-v11-release: true,
 );
 
+@use 'sass:meta';
```
Comment (tw15egan) [inline packages/carbon-react/.storybook/styles.scss]:
```suggestion
```

File: packages/styles/scss/components/tag/_tag.scss [hunk 3]
```diff
@@ -0,0 +1,269 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use 'sass:meta';
```
Comment (tw15egan) [inline packages/styles/scss/components/tag/_tag.scss]: Is this needed?

**Comment by tw15egan** [comment]
@dakahn just a few comments, but it looks great! 🎉 🎸

**Review by andreancardona — COMMENTED** [review]
approved - pending TJ's comments 🌮

File: packages/carbon-react/.storybook/styles.scss [hunk 4]
```diff
@@ -23,18 +27,22 @@ $feature-flags: (
 
 :root {
   @include styles.theme(styles.$white, button.$white);
+  @include styles.theme(styles.$white, tag.$white);
```
Comment (joshblack) [inline packages/carbon-react/.storybook/styles.scss]:
```suggestion
  @include styles.theme(styles.$white, button.$white, tag.$white);
```

File: packages/carbon-react/src/components/Tag/Tag.stories.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Tag } from 'carbon-components-react';
+
+export default { title: 'Components/Tag' };
```
Comment (joshblack) [inline packages/carbon-react/src/components/Tag/Tag.stories.js]: I think we can add in a `component: Tag` field to this for the prop table to work

**Review by joshblack — CHANGES_REQUESTED** [review]

**Review by joshblack — APPROVED** [review]

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Pull request merged**